src/main/java/sg/ncl/MainController.java

```diff
@@ -1426,14 +1425,26 @@ public String validateExperiment(
             return "redirect:/experiments/create";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "redirect:/experiments/create" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)